common/api-review/data-connect.api.md

```diff
@@ -109,16 +109,6 @@ export interface DataConnectResult<Data, Variables> extends OpResult<Data> {
     ref: OperationRef<Data, Variables>;
 }
 
-// @public
-export interface DataConnectSubscription<Data, Variables> {
```

Comment:
Note: this was removed as it's an internal interface wasn't being referenced by any of our public api functions